frontend/src/components/abfragevarianten/AbfragevarianteFormular.vue

```diff
@@ -329,6 +330,9 @@ export default class AbfragevarianteForm extends Mixins(
   @Prop({ type: Boolean, default: false })
   private readonly isEditable!: boolean;
 
+  @Prop()
+  private anzeigeContextAbfragevariante: AnzeigeContextAbfragevariante | undefined;
```

Comment:
Da es `AnzeigeContextAbfragevariante.UNDEFINED` gibt, halte ich das `| undefined` für überflüssig.

Reply:
Das Problem ist, dass ein Defaultwert bei Enum-Properties relativ komplexe Workarounds benötigt, die man sich mit einem potentiellen Undfined sparen kann.

https://github.com/kaorun343/vue-property-decorator/issues/202

Habe es jetzt so gelöst:

![image](https://github.com/it-at-m/isi-frontend/assets/56300801/1176c722-e7e4-45bb-9e00-56d2ff98e029)